shared/js/initApp.js

```diff
@@ -1,157 +1,116 @@
 // TODO (brent) - way too many globals
 // TODO (brent) - I wonder if we should sub-namespace dashboard
-/* global script_path, Dialog, CDOSounds, dashboard, appOptions, $, trackEvent, Applab, sendReport, cancelReport, lastServerResponse, showVideoDialog, ga*/
+/* global script_path, Dialog, CDOSounds, dashboard, appOptions, $, trackEvent, Applab, sendReport, cancelReport, lastServerResponse, showVideoDialog, ga, digestManifest*/
 
 var timing = require('./timing');
 var chrome34Fix = require('./chrome34Fix');
-dashboard.project = require('./project');
 
-if (!window.dashboard) {
-  throw new Error('Assume existence of window.dashboard');
-}
+window.apps = {
+  load: require('./loadApp'),
```

Comment:
I would prefer to see these requires (loadApp and project) at the top of the file as well.